entity-framework/core/providers/cosmos/functions.md

```diff
@@ -11,82 +11,82 @@ This page shows which .NET members are translated into which SQL functions when
 
 ## Date and time functions
 
-.NET                  | SQL
---------------------- | ---
-DateTime.UtcNow       | GetCurrentDateTime()
-DateTimeOffset.UtcNow | GetCurrentDateTime()
+| .NET                  | SQL                  |
```

Comment:
Any reason to have the closing vertical line on the right side of the table? Seems like needless extra management

Reply:
It's the way Rider formats tables, and makes them easier to read. Shouldn't be any extra work if you're using Rider, and I presume and decent IDE/editor has similar functionality.